go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/apache/dubbo-getty
 go 1.14
 
 require (
-	github.com/dubbogo/gost v1.11.19
+	github.com/dubbogo/gost v1.11.20-0.20211118050323-7eddeade71e1
```

Comment:
why not release gost a new version first?

Reply:
i need check the standard of the tag

Reply:
address